openpype/hosts/fusion/api/pipeline.py

```diff
@@ -39,12 +42,13 @@
 INVENTORY_PATH = os.path.join(PLUGINS_DIR, "inventory")
 
 
-class CompLogHandler(logging.Handler):
+class FusionLogHandler(logging.Handler):
+    # Keep a reference to fusion's Print function (Remote Object)
+    _print = getattr(sys.modules["__main__"], "fusion").Print
+
     def emit(self, record):
         entry = self.format(record)
-        comp = get_current_comp()
-        if comp:
-            comp.Print(entry)
+        self._print(entry)
```

Comment:
This is mostly an optimization.

Reply:
**Warning:** I've just found that this logic does not work in Fusion 17.2 (released around June 2021) but does work in Fusion 17.4 since the `fusion` instance does not have a `Print` method in the older Fusion.

We might need to introduce a fallback to support older fusion versions.